drivers/SmartThings/bose/config.yml

```
@@ -3,3 +3,5 @@ packageKey: 'bose'
 permissions:
   lan: {}
   discovery: {}
+description: "Smartthings driver for Bose devices"
```

```suggestion
description: "SmartThings driver for Bose devices"
```
Here and everywhere else